chains/bob-sepolia-testnet.json

```diff
@@ -0,0 +1,25 @@
+{
+  "alias": "bob-sepolia-testnet",
+  "blockTimeMs": 503,
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://testnet-explorer.gobob.xyz/api"
+    },
+    "browserUrl": "https://testnet-explorer.gobob.xyz/"
+  },
+  "id": "111",
+  "name": "Bob Sepolia testnet",
+  "providers": [
+    {
+      "alias": "default",
+      "rpcUrl": "https://testnet.rpc.gobob.xyz/"
+    }
+  ],
+  "skipProviderCheck": true,
+  "symbol": "ETL",
```

Comment:
ETH?

> BOB is EVM-based and we recommend using the [Solidity](https://docs.soliditylang.org/en/v0.8.24/) language to write your smart contracts. The base currency on BOB is ETH. Since BOB is a rollup on Ethereum, you can bridge over ETH from Ethereum to pay for transactions.

Taken from [here](https://docs.gobob.xyz/docs/build/getting-started/helloworld).